### ISSUE TITLE
Template: Images require alt text tags

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#79]
Spacer images in the footer do not have alternative text attributes.

all "img" elements should have an 'alt' attribute with 'alt=""'

From [1.1.1 Non-Text Context](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content.html)
